2-Pricing/app.css

```diff
@@ -0,0 +1,355 @@
+:root {
+    --spacing: 1.875rem;
+    --featured-color: #EA580C;
+    --bravo-color: #EAB308;
+    --charlie-color: #22C55E;
+    --extra-color: #EF4444;
+}
+
+body {
+    background: no-repeat url("./images/Italian-streets.png");
+    background-size: cover;
+    background-position: center;
+    font-family: 'Manrope', sans-serif;
+    color: white;
+}
+
+#cart {
+    background: none;
+    border: none;
+    position: fixed;
+    top: 2rem;
+    right: 0.1rem;
+    transition: 0.2s ease-in-out;
+    z-index: 10;
+}
+
+#cart:hover { transform: scale(1.2); }
+
+main {
+    min-width: 18.75rem;
+    min-height: 100vh;
+    background-color: rgba(0, 0, 0, 0.75);
+}
+
+.view {
+    display: flex;
+    align-items: center;
+    border-radius: 4rem;
+    border: solid 1px white;
+}
+
+a {
+    font-size: 0.625rem;
+    padding: 0 0.5rem;
+    text-decoration: none;
+    color: white;
+    transition: all 0.35s ease-in-out;
+}
+
+h2 { margin: 0;}
+
+.selected-view {
+    color: black;
+    background-color: white;
+    font-weight: bold;
+}
+
+.view-left {
+    border-radius: 4rem 0 0 4rem;
+}
+
+.view-right {
+    border-radius: 0 4rem 4rem 0;
+}
+
+
+.container, .heading-section, .subscriptions,
+.card, .card-top, .card-bottom, .oval {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
```

Comment:
I think it would be more semantic to define a `flex-column` class and apply it to all these elements, instead of adding these 4 props to all those 7 classes. What do you think?